sdk/storage/Cargo.toml

```diff
@@ -43,6 +43,7 @@ tokio = { version = "0.2", features = ["macros"] }
 env_logger = "0.7"
 azure_identity = { path = "../identity" }
 url = "2.1"
+reqwest = "*"
```

Comment:
Can we switch to a non "*" version?

Reply:
Ah yes of course, it's just me being lazy 🤕, but it should be removed anyway ASAP.

The goal here is to remove the dependency from `reqwest` and `hyper` altogether.  The `storage` crate (and the other ones as well) will only accept the `HttpClient` trait (defined in `core`). They should not know what kind of http client implementation the user chooses. 

Also we could/should gate the support for `hyper`, `reqwest`, etc... under features (in core) so we could avoid the dependency if not needed.

Reply:
Fixed!